src/components/flyout/Flyout.scss

```diff
@@ -32,7 +32,7 @@
             transform: none !important;
         }
 
-        .overlay {
+        & > .flyout-overlay > .overlay {
```

Comment:
wouldn't this be confusing that there are two `flyout-overlay` classes on separate elements?